sympy/physics/vector/point.py

```diff
@@ -483,19 +483,46 @@ def vel(self, frame):
         Examples
         ========
 
-        >>> from sympy.physics.vector import Point, ReferenceFrame
+        >>> from sympy.physics.vector import Point, ReferenceFrame, dynamicsymbols
         >>> N = ReferenceFrame('N')
         >>> p1 = Point('p1')
         >>> p1.set_vel(N, 10 * N.x)
         >>> p1.vel(N)
         10*N.x
+        >>> p = Point('p')
```

Comment:
```suggestion

Velocities will be automatically calculated if possible, otherwise a ``ValueError`` will be returned.

        >>> p = Point('p')
```

Reply:
Line 493 is to be added as comment, right?